src/commons/api/history.rs

```diff
@@ -770,6 +775,11 @@ impl fmt::Display for StorableCaCommand {
                 write!(f, "Remove ASPA for customer AS: {}", customer)
             }
 
+            // ------------------------------------------------------------
+            // BGPSec Support
+            // ------------------------------------------------------------
+            StorableCaCommand::BgpSecDefinitionUpdates => write!(f, "Update BGPSec definitions"),
```

Comment:
That is because the original command here contains definitions for which we may reject the CSR later if we become more picky in future. That said, all relevant details are in the saved events and they are visible there.